### ISSUE TITLE
Prettier console output

### ISSUE THREAD

**Issue by dariooddenino** [joelmccracken/reddup#19]
Hi!
Right now it's a bit hard to figure out what's going on since the output is uniform white text without any separation. 
It would be great to have colors / newlines / symbols to help read the output more easily :smile:

**Comment by joelmccracken** [comment]
Good idea. Maybe we can also post examples and perhaps what might be better output for them in this thread?
